lib/board.rb

```diff
@@ -0,0 +1,56 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    self.display
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(user_input)
+    index = user_input.to_i - 1
+    @cells[index]
+  end
+
+  def full?
+    @cells.include?(" ") == false
+  end
+
+  def turn_count
+    9 - @cells.count(" ")
```

Comment:
Consider this alternative using Ruby's [`#count`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-count). Entire body could read:

```ruby
def turn_count
  cells.count { |cell| cell == "X" || cell == "O" }
end
```